main.go

```diff
@@ -53,20 +58,28 @@ func main() {
 		glog.Fatalf("Error building kubernetes clientset: %s", err.Error())
 	}
 
-	exampleClient, err := clientset.NewForConfig(cfg)
+	controllerClient, err := clientset.NewForConfig(cfg)
```

Comment:
Renamed `exampleClient` as I don't think it's really an appropriate name. Is there a better name than `controllerClient`?

Reply:
> Renamed exampleClient as I don't think it's really an appropriate name. Is there a better name than controllerClient?

Your controller may have many clients.  I'd suggest naming them for their target groups.  This one is your `etcdstorageClient`, right?

Reply:
The controller is [called `etcdproxy` for now](https://github.com/xmudrii/etcdproxy-controller/pull/2/files#diff-e2f580b65ff5eae476b26f201aee4397R51), so let's call client like that as well. I'm going to bring up discussion about controller reorganization soon, so we're going to talk  more about this as well.